chromium_src/third_party/blink/renderer/modules/websockets/websocket_channel_impl.h

```diff
@@ -30,7 +30,7 @@ using WebSocketChannelImpl_BraveImpl = WebSocketChannelImpl;
 
 namespace blink {
 
-class MODULES_EXPORT WebSocketChannelImpl
+class MODULES_EXPORT WebSocketChannelImpl final
```

Comment:
here are follow up fixes from the original PR https://github.com/brave/brave-core/pull/11609